src/error_temp.c

```diff
@@ -0,0 +1,60 @@
+
+#include "mod_executor.h"
+
+/**
+ * @brief Print warning messages depending on 
+ * received indicator and trigger string.
+ * 
+ * @param indic		Indicator string for error message.
+ * @param trigger	Trigger string causing the error.
+ * @return t_err	ERR_HEREDOC_EOF, ERR_DIR, SUCCESS
+ */
+t_err	ft_print_warning(char *indic, char *trigger)
```

Comment:
Why not just use t_err to differentiate the errors? Would get rid of the strncmp() calls.

(If err == ERR_HEREDOC_EOF)
print eof message

Reply:
Cause not very error has a unique ERR_. This is due to the implementation with ft_check_cmd_access() being followed directly by ft_process_cmd() which takes the returned err to determine its actions. 

As indicated this might be refactored later on when the parent functions ft_execute_scmd() and ft_execute_pcmd() are altered for error handling. A cleaner differentiation between ft_check_dir(), ft_check_access would help this regarding.

Reply:
I think you already refactored a bit: I currently find four instances of print_warning(). Each in direct response to an error.